### DISCUSSION TITLE
How to donate??

### DISCUSSION THREAD

**Discussion by 1456319** [Woolverine94/biniou#43] [General]
paypal preferred, or buymeacoffee

i dont see any links on the main page, sorry if its there

**Comment by Woolverine94** [comment]
Hello @1456319 ,

Simple answer : You can't ! 

biniou is a project that heavily rely on the work of others developers. 
I don't want to start a patreon for biniou, cause I don't think it will be fair for all the devs that do the really hard work (mine is pretty simple in comparison !), and invest money in their projects -which is not my case : I only invest time-. 
Also, I have a full time job that has nothing to do with biniou and I don't really need to be sponsored.

So, if you want to support the project by a donation, make it to the models creators or other github projects I rely on. Some of them really need and deserve it, way more than me ! 

You can find the list of used github projects at the end of the README.md and you can find the models references in the "Model list" of each module (copy/paste the models name in google, it should display the huggingface page of the mdoel as the first result). 

Thanks for them and your kind proposal !

**Reply by 1456319** [reply]
I see. I will support their projects in this case.

 I hope you don't get bored with this hobby - no matter how I look at this (I'm not a coder), biniou appears to be in a league of its own. To lose it because you no longer have the time to support it would be tragic.

Much respect,  Best of luck!

**Reply by Woolverine94** [reply]
Thanks for your support and your kind message ! 

I'll try to maintain biniou as long as I can, but it's also heavily depending on others projects and I don't have any kind of control on them, so I can't promise you it will last forever.
